src/ImageRunModal.jsx

```diff
@@ -218,13 +218,22 @@ export class ImageRunModal extends React.Component {
 
     getCreateConfig() {
         let createConfig = {};
+        // HACK: checking version is because API got broken from 1.2 -> 1.3
+        // Once we support only 1.3 podman, commit which introduces these changes can be reverted
 
-        createConfig.image = this.state.image.repoTags ? this.state.image.repoTags[0] : "";
+        if (this.props.version < "1.3.0")
```

Comment:
You can't compare version numbers lexicographically. It will break as soon as you have components in the version that are larger than 10. For example, `"1.10.0" < "1.3.0"` evaluates to `true`, even though the first version is larger.

There's a `semver` package to do this, but that might be overkill for this little check. You could either split the string and compare the components in order, or skip the version check completely and check the type of the `repoTags` you receive directly (the API difference is that it used to return an array, right?).